pages/events/event_edit.php

```diff
@@ -0,0 +1,58 @@
+<?php
+restrict_access("ROOT", "STAFF", "COACH", "COACHSTAFF");
+
+require_once "database/events.api.php";
+
+$event_id = get_route_param("event_id", false);
+$post = $_POST;
+if ($event_id) {
+    $event = get_event_by_id($event_id, $_SESSION['user_id']);
+    if (empty($_POST)) {
+        $post = [
+            "event_name" => $event["nom"],
+            "start_date" => $event["depart"],
+            "end_date" => $event["arrivee"],
+            "limit_date" => $event["limite"]
+        ];
+    }
+}
+
+$v = validate($post);
+$event_name = $v->string("event_name")->label("Nom de l'événement")->placeholder()->required();
+$start_date = $v->date("start_date")->label("Date de départ")->required();
+$end_date = $v->date("end_date")->label("Date de retour")->required()->after($start_date->value);
```

Comment:
Maybe we should forbid the creation of events if any of the dates is before the request date?

Reply:
Good point. I will also fix being allowed to set the limit date the same day as the event itself 🙈

Reply:
Done!